pkg/graveler/errors.go

```diff
@@ -59,6 +59,7 @@ var (
 	ErrTooManyTries                 = errors.New("too many tries")
 	ErrSkipValueUpdate              = errors.New("skip value update")
 	ErrImport                       = wrapError(ErrUserVisible, "import error")
+	ErrReadOnlyRepository           = errors.New("read-only repository")
```

Comment:
```suggestion
	ErrReadOnlyRepository           = wrapErrors(ErrUserVisible, "read-only repository")
```
or reporting it becomes somewhat harder.